modules/test-runner/src/deposit/deposit.test.ts

```diff
@@ -68,7 +68,11 @@ describe("Deposits", () => {
       client: ONE,
       assetId: AddressZero,
     };
-    await client.deposit({ amount: expected.client, assetId: expected.assetId });
+    const response = await client.deposit({ amount: expected.client, assetId: expected.assetId });
+    if (response.completed) {
+      expect(response.txHash).to.be.ok;
```

Comment:
Would it be feasible for client.deposit to return the entire transaction instead of just the hash? bc that'd frequently save an `await ethProvider.getTransaction(txHash)` step

Reply:
Yes this is definitely doable.